rules/no-unnecessary-polyfills.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+const semver = require('semver');
+const readPkgUp = require('read-pkg-up');
+const compatTable = require('@babel/preset-env/data/built-ins.json');
+const upperfirst = require('lodash.upperfirst');
+const camelcase = require('lodash.camelcase');
+const getDocsUrl = require('./utils/get-docs-url');
+
+function isRequireCall(node) {
```

Comment:
This method is misleading as it doesn't actually check it's a method. Better to just inline it where it's used.